packages/wrangler/turbo.json

```diff
@@ -21,55 +21,61 @@
 				"config-schema.json"
 			],
 			"env": [
-				"NODE_EXTRA_CA_CERTS",
-				"CLOUDFLARE_ACCOUNT_ID",
-				"SOURCEMAPS",
-				"NODE_ENV",
-				"SPARROW_SOURCE_KEY",
 				"ALGOLIA_APP_ID",
 				"ALGOLIA_PUBLIC_KEY",
-				"CLOUDFLARE_API_TOKEN",
-				"CLOUDFLARE_ACCOUNT_ID",
-				"WRANGLER_AUTH_DOMAIN",
-				"PATH",
-				"WRANGLER_LOG",
-				"EXPERIMENTAL_MIDDLEWARE",
-				"FORMAT_WRANGLER_ERRORS",
+				"NODE_ENV",
+				"SOURCEMAPS",
+				"SPARROW_SOURCE_KEY",
+				"SENTRY_DSN"
+			],
+			"passThroughEnv": [
+				"CF_PAGES_UPLOAD_JWT",
 				"CF_PAGES",
-				"WORKERS_CI",
 				"CI",
-				"CF_PAGES_UPLOAD_JWT",
+				"CLOUDFLARE_ACCOUNT_ID",
+				"CLOUDFLARE_API_TOKEN",
+				"CUSTOM_BUILD_VAR",
 				"EXPERIMENTAL_MIDDLEWARE",
+				"FORMAT_WRANGLER_ERRORS",
+				"http_proxy",
+				"HTTP_PROXY",
+				"https_proxy",
+				"HTTPS_PROXY",
+				"HYPERDRIVE_DATABASE_URL",
+				"LC_ALL",
 				"NO_D1_WARNING",
 				"NO_HYPERDRIVE_WARNING",
-				"WRANGLER",
-				"WRANGLER_IMPORT",
-				"CUSTOM_BUILD_VAR",
+				"PATH",
 				"PWD",
-				"LC_ALL",
-				"WRANGLER_SEND_METRICS",
-				"https_proxy",
-				"HTTPS_PROXY",
-				"http_proxy",
-				"HTTP_PROXY",
-				"CI_OS",
-				"SENTRY_DSN",
 				"SYSTEMROOT",
 				"TZ",
+				"WORKERS_CI",
+				"WRANGLER_API_ENVIRONMENT",
+				"WRANGLER_AUTH_DOMAIN",
 				"WRANGLER_DISABLE_EXPERIMENTAL_WARNING",
 				"WRANGLER_DISABLE_REQUEST_BODY_DRAINING",
-				"WRANGLER_WORKER_REGISTRY_PORT",
-				"WRANGLER_API_ENVIRONMENT",
-				"HYPERDRIVE_DATABASE_URL"
+				"WRANGLER_LOG",
+				"WRANGLER_SEND_METRICS",
+				"WRANGLER_WORKER_REGISTRY_PORT"
 			]
 		},
 		"test:ci": {
 			"inputs": ["!*/**/.wrangler/**", "**/__tests__/**"],
-			"dependsOn": ["build"]
+			"dependsOn": ["build"],
+			"env": ["VITEST", "NODE_DEBUG", "MINIFLARE_WORKERD_PATH"]
```

Comment:
These env vars affect the running of the tests. They need to be in the cache key but also if they were not here they would not be provided to the underlying test process so would have no effect.